about-us.php

```diff
@@ -1,98 +1,72 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-        <title>About Us</title>
-        <link href="https://fonts.googleapis.com/css2?family=Reggae+One&display=swap" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css2?family=Padauk&family=Reggae+One&display=swap" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css2?family=Cabin+Sketch&display=swap" rel="stylesheet">
-</head>
-<style> 
-    body {
-    background-color: gray;
-  }
-    h1 {
-    font-family: 'Reggae One', cursive;
-  }      
-  p { 
-    font-family: 'Padauk', sans-serif;
-  }
-  h2 {
-    font-family: 'Cabin Sketch', cursive;
-  }
-
-</style>
-
-<?php include('nav-bar.php'); ?>
+<!doctype html>
+<html lang="en">
+  <head>
+    
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <title>Behind the Reel</title>
+  </head>
+  
 
 <body>
-        
-        <div>
+<?php include('nav-bar.php'); ?>
+    <div class="bannerImage">
+        <img src="crispy-images/indiefilms.jpg">
+    </div>
+    
+    <div class="container">  
+        <section>
+            <div class="pageTitle">
             <h1>About Us</h1>
-
-            <p>
-                <br>Beyond the Reel is a visual production company. Specializing in events, music videos, 
-                <br>commercials and short films.  
-                <br>Over the last decade, we have brought over 100 projects to life for an ever 
-                <br>expanding clientele base.  
-                <br>Our passion for capturing unforgettable moments drives us to continue to be 
-                <br>leaders within our industry.  
+            </div>    
+        </section>
```

Comment:
This seems overly complex. Why does an h1 have both a section and its own div?

Reply:
Removed section and div for heading - Commit #1d26e3d